src/components/page/page.module.scss

```diff
@@ -53,4 +54,14 @@
   .grid-media {
     grid-area: grid-media;
   }
```

Comment:
probably to go along with our ditching `MediaTag` in the components, let's change these `grid-media`s to `grid-widget`s

Reply:
done and done